components/builder-api/src/server/resources/channels.rs

```diff
@@ -58,38 +45,6 @@ impl Channels {
                 "/depot/channels/{origin}/{channel}",
                 Method::DELETE,
                 delete_channel,
-            ).route(
```

Comment:
I suggest we leave these routes here and not break up channels without breaking up the route prefix as well - each resource should have all routes at the same prefix in one place